language/resource.language.sk_sk/strings.po

```diff
@@ -584,113 +1313,1281 @@ msgctxt "#31321"
+msgid "Reset skin colors to default"
+msgstr "Vrátiť farby vzhľadu na predvolené hodnoty"
+
+msgctxt "#31497"
+msgid "Move the system bar to the top of the home window. Only works when the rss feed is not enabled or in the high position."
+msgstr "Presun systémovej lišty do hornej časti domovskej obrazovky. Funguje len vtedy, ak nie je spravodajstvo RSS umiestnené na vrchu obrazovky."
+
+msgctxt "#31498"
+msgid "Move System Bar to top of screen"
+msgstr "Presunúť systémovú lištu na vrch obrazovky"
+
+msgctxt "#31499"
+msgid "Smaller list"
+msgstr "Menší zoznam"
+
+msgctxt "#31500"
+msgid "This will set the music discart fallback to display a vinyl record instead of a blank CD."
+msgstr "V prípade nenájdenie grafiky pre hudobné CD sa použije obrázok vinylovej platne namiesto prázdneho CD."
+
+msgctxt "#31501"
+msgid "Channel group"
+msgstr "Skupina kanálov"
+
+msgctxt "#31502"
+msgid "Live TV"
+msgstr "Televízia"
+
+msgctxt "#31503"
+msgid "Add Group"
+msgstr "Pridať skupinu"
+
+msgctxt "#31504"
+msgid "Rename Group"
+msgstr "Premenovať skupinu"
+
+msgctxt "#31505"
+msgid "Delete Group"
+msgstr "Odstrániť skupinu"
+
+msgctxt "#31506"
+msgid "Available[CR]Groups"
+msgstr "Dostupné[CR]skupiny"
+
+msgctxt "#31507"
+msgid "Show clearart in tv information"
+msgstr "Zobraziť clearart v informáciách o TV"
+
+msgctxt "#31508"
+msgid "Move OSD buttons to top"
+msgstr "Presunúť tlačidlá OSD na vrch obrazovky"
+
+msgctxt "#31509"
+msgid "Tiny posters"
+msgstr "Úplne malé plagáty"
+
+msgctxt "#31510"
+msgid "JukeBox"
+msgstr "JukeBox"
+
+msgctxt "#31511"
+msgid "Channel Options"
+msgstr "Možnosti kanálu"
+
+msgctxt "#31512"
+msgid "Enable fullscreen information for recordings"
+msgstr "Povoliť informácie na celej obrazovke pre nahrávky"
+
+msgctxt "#31513"
+msgid "Enable fullscreen information for add-ons"
+msgstr "Povoliť informácie na celej obrazovke pre doplnky"
+
+msgctxt "#31514"
+msgid "Resource image addon for displaying the coloured studio icons throughout the skin."
+msgstr "Doplnok s kolekciou obrázkov pre zobrazovanie farebných ikon filmových štúdií v celom vzhľade."
+
+msgctxt "#31515"
+msgid "Enable language flags for subtitles, audio language or both to the mediaflags. Note that some views will not display them"
+msgstr "Pridá ikony jazykov pre titulky a zvukové stopy k značkám médii. Poznámka: Nie všetky typy zobrazení ich zobrazujú."
+
+msgctxt "#31516"
+msgid "Disable this to show a simplified display of lyrics instead of the default \"Karaoke\" style"
+msgstr "Zakážte túto voľbu, ak si prajete jednoduché zobrazovanie textov piesní a nie v štýle \"karaoke\""
+
+msgctxt "#31517"
+msgid "Music (Artists)"
+msgstr "Hudba (Umelci)"
+
+msgctxt "#31518"
+msgid "Music (Albums)"
+msgstr "Hudba (Albumy)"
+
+msgctxt "#31519"
+msgid "Party mode (Music videos)"
+msgstr "Párty režim (Videoklipy)"
+
+msgctxt "#31520"
+msgid "Movies in set"
+msgstr "Filmy v kolekcii"
+
+msgctxt "#31521"
+msgid "More movies starring"
+msgstr "More movies starring"
```

Comment:
Currently not used but may be in the future. Want to translate it now?

Reply:
Now I'm not quite sure. If it's related to the actor and his cast in other movies, this is the translation:
```suggestion
msgstr "Ďalšie účinkovanie"
```